apps/miniapp/index.js

```diff
@@ -158,7 +158,7 @@ class CondoOIDCMiddleware {
                 delete req.session[oidcSessionKey]
                 await req.session.save()
 
-                return res.redirect('/')
+                return res.json({ user: { id: user.id } })
```

Comment:
Callback without any redirect?

Reply:
That's how mobile flow works right now. When you call auth from `cordova` plugin, something similar to webview is opened (but hidden), where v1 cookies are set. After this it follows auth flow (`miniapp/oidc/auth` -> `condo/oidc/auth/` ->  ... -> `miniapp/oidc/callback`) and returns response from latest server to original callback, so you can resolve it on client.

While processing, you can show loader or suggest alternative client route in case of no internet access.

We wanted to merge this flow with web via `bridge`, so you can write you auth uniformly for B2B and B2C apps